chainguard-dev/edu
# Enforce IA - moving non-automated Enforce docs [PLEASE MERGE IF YOU WANT]

## Type of change

This is my first solid try at moving around the enforce docs. ~This PR may get deleted quickly if the aliases don't work (they weren't working on my local testing).~ They work!

### What should this PR do?

resolves #521 and #534 

### Why are we making this change?

Let's make the docs look better!

### What are the acceptance criteria? 


I made some executive decisions regarding the organization. I'm very open to making updates as necessary.

### How should this PR be tested?

Confirm that aliases work and everything is laid out to your liking! 

EDIT: Be aware that the netlify warning about a redirect rule failing seems to be in error. Feel free to slack me if you have questions.
